T21694a: Urkh! But expected!
CallStack (from HasCallStack):
  error, called at T21694a.hs:24:33 in main:Main
HasCallStack backtrace:
CallStack (from HasCallStack):
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:89:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:125:3 in ghc-internal:GHC.Internal.Exception


